php/php.dbgp/src/org/netbeans/modules/php/dbgp/packets/Property.java

```diff
@@ -64,12 +66,9 @@ public String getName() {
     }
 
     public void setName(String value) {
-        Node node = getNode().getAttributes().getNamedItem(NAME);
-        if (node == null) {
-            node = getNode().getOwnerDocument().createAttribute(NAME);
-            getNode().appendChild(node);
-        }
-        node.setNodeValue(value);
+        assert getNode() instanceof Element;
+        Element element = (Element) getNode();
```

Comment:
Please check the type to avoid the ClassCastException.

Reply:
Check added.
Hopefully it was what you meant.

Reply:
One note to the `assert` - imagine that it would happen - what it will take tell you? Just that the node is not an `Element`... Not great, probably, and the assertion would need to be likely improved to something like:

```
assert getNode() instanceof Element : "expected Element but got " + getNode().getClass().getName();
```